aspnetcore/tutorials/razor-pages/uploading-files.md

```diff
@@ -55,17 +55,36 @@ To avoid code duplication for processing uploaded schedule files, add a static h
 
 ### Save the file to disk
 
-The sample app saves the file's content into a database field. To save the file's content to disk, use a [FileStream](/dotnet/api/system.io.filestream):
+The sample app saves uploaded files into database fields. To save a file to disk or a network share, use a [FileStream](/dotnet/api/system.io.filestream). The following example copies a file held by `FileUpload.UploadPublicSchedule` to a `FileStream` in an `OnPostAsync` method. The `FileStream` writes the file to disk or a network share at the `<PATH-AND-FILE-NAME>` provided:
```

Comment:
Avoid network shares in an example, by default app pools are local accounts which can't access network shares. Instead maybe D:\uploadedFiles?